src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html

```diff
@@ -905,10 +905,7 @@ <h5 class="col-sm-2">
 
 
     
-        <div id="statusMessage" class="alert alert-warning">
-            The feedback session has been updated.
-        </div>
-        <script type="text/javascript" src="/js/statusMessage.js"></script>
+        <div id="statusMessage" style="display: none;"></div>
```

Comment:
When we edit `textarea`, the content is not updated in html page source so the `verifyMainHtml` will fail. I want to check the new content in `textarea` so I refresh the page in the test before calling `verifyMainHtml`.